service/service_test.go

```diff
@@ -126,43 +158,12 @@ func TestRun(t *testing.T) {
 			})
 		})
 
-		/* ADD CODE OR REMOVE: put this code in, if you have Checkers to register
```

Comment:
Why has this been removed? There should be checkers registered for the mongo client so I would assume this is OK to leave in?

Reply:
thank you. I put it back in.

Reply:
Thank you. Added code.